src/standard/effectBuilder.html

```diff
@@ -168,7 +168,8 @@
 
     // method expressions are of the form: `name([arg1, arg2, .... argn])`
     _parseMethod: function(expression) {
-      var m = expression.match(/(\w*)\((.*)\)/);
+      // tries to match valid javascript property names
+      var m = expression.match(/([a-zA-Z_$][0-9a-zA-Z_$]*)\((.*)\)/);
```

Comment:
Why not just `/([\w\d$]+)\((.*)\)/`?
That will capture more valid cases, including functions that have to be quoted in the prototype.